packages/cli/src/commands/pg/locks.ts

```diff
@@ -0,0 +1,56 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import {database} from '../../lib/pg/fetcher'
+import {exec} from '../../lib/pg/psql'
+import heredoc from 'tsheredoc'
+
+export default class Locks extends Command {
+  static topic = 'pg'
+  static description = 'display queries with active locks'
+  static flags = {
+    truncate: flags.boolean({char: 't', description: 'truncates queries to 40 characters'}),
+    app: flags.app({required: true}),
+    remote: flags.remote(),
+  }
+
+  static args = {
+    database: Args.string(),
+  }
+
+  public async run(): Promise<void> {
+    const {flags, args} = await this.parse(Locks)
+    const {app, truncate} = flags
+    const db = await database(this.heroku, app, args.database)
+    const query = heredoc`
+      SELECT
+        pg_stat_activity.pid,
+        pg_class.relname,
+        pg_locks.transactionid,
+        pg_locks.granted,
+        ${this.truncatedQueryString(truncate)} AS query_snippet,
+        age(now(), pg_stat_activity.query_start) AS "age"
+      FROM
+        pg_stat_activity,
+        pg_locks
+          LEFT OUTER JOIN pg_class
```

Comment:
nice job fixing the format of this query. i was trying to find this part in the original command and saw the weird dangling `left`